src/autobind.cr

```diff
@@ -56,6 +73,15 @@ while arg = ARGV[i += 1]?
     while a fixed value will remove just that:
         --remove-enum-prefix[=true,false,<value>]
         --remove-enum-suffix[=true,false,<value>]
+
+    The resulting library name can be controlled with the --lib-name argument,
+    and it can be wrapped in a parent module with --module-name.
+        --lib-name="LibC"         (the default)
+        --module-name="Library"
+        --mod "Library"           (also ok)
+        --lib "LibC"              (this too)
```

Comment:
No need to add aliases, I would choose `--lib-name=` and `--parent-module=`

Reply:
If that's what you prefer.